fastlane/metadata/android/en-US/changelogs/930.txt

```diff
@@ -0,0 +1,19 @@
+New
+• Search on YouTube Music
+• Basic Android TV support
+
+Improved
+• Added the ability to remove all watched videos from a local playlist
+• Show message when content isn't supported yet instead of crashing
+• Improved popup player resize with pinch gestures
+• Enqueue streams on long press on background and popup buttons in channel
+• Improved size handling of the drawer header title
+
+### Fixed
```

Comment:
Why does this have ### before it and the others not? Also, why do we use bullet points in some changelogs while using hyphens in others?

Reply:
Thanks, I forgot to remove the ###.
Well, F-Droid's website supports markdown, while the app does not. I didn't knew this when I wrote the first chagelogs. That's why I switched to UTF-8 characters and used .txt files instead.